### ISSUE TITLE
Orange used in nick coloring is very light

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#8500]
Consider using a darker orange in the set of available nick colors; the current #ffc666 is hard to read on a white background

**Comment by turt2live** [comment]
Original source: https://matrix.to/#/!iNmaIQExDMeqdITdHH:matrix.org/$15497759362266487pcXFm:matrix.org?via=matrix.org&via=linuxgaming.life&via=disroot.org

**Comment by lampholder** [comment]
![image](https://user-images.githubusercontent.com/1922197/53011317-06e51480-3438-11e9-8425-d50d7a01f5a0.png)

And the WCAG AA verdict is... FAIL.
![image](https://user-images.githubusercontent.com/1922197/53011402-2bd98780-3438-11e9-9695-5237f554f8d0.png)

**Comment by kittykat** [comment]
Orange is now #ff812d, which is darker
